website/i18n/en.json

```diff
@@ -341,9 +341,6 @@
       "toastandroid": {
         "title": "ToastAndroid"
       },
-      "toolbarandroid": {
```

Comment:
@ronnyiscoding Same problem here.

Reply:
PR updated here https://github.com/facebook/react-native-website/pull/1590/commits/12166a4f8a16a7bd82ec553fb7675fd0886fe5d2 thanks!

Reply:
Thank you for the changes. 

As @rachelnabors said (which apply for the both PRs) you should be updating:
* `master` docs (`docs/activityindicator.md`) 

not:

* `versioned` docs (`website/versioned_docs/version-0.61/activityindicator.md`)

Reply:
Ok, Thank you, PR updated at https://github.com/facebook/react-native-website/pull/1590/commits/51913e47619c9295d44ffe4543601b6335e2d541